src/view/com/profile/ProfileHeader.tsx

```diff
@@ -234,7 +238,7 @@ let ProfileHeaderLoaded = ({
 
   const onPressShare = React.useCallback(() => {
     track('ProfileHeader:ShareButtonClicked')
-    shareUrl(toShareUrl(makeProfileLink(profile)))
+    ShareUrl(toShareUrl(makeProfileLink(profile)))
```

Comment:
Why was this function capitlalized? Usually that is reserved for React Components.

Reply:
It was reverted, thanks.